scripts/load_test_ntc_server.js

```diff
@@ -0,0 +1,255 @@
+import fs from 'fs';
+import { WebSocket } from 'ws';
+import BinaryFrame from '../public/js/BinaryFrame.js';
+import { shuffle } from '../public/views/utils.js';
+// import { Worker, isMainThread, workerData } from 'worker_threads';
```

Comment:
If we're using `cluster.fork()` instead of worker threads, just remove the code rather than commenting it out.